File: 709 - To Lower Case.py
class Solution:
    def toLowerCase(self, str: str) -> str:
        t = [i for i in str]
        for i in range(len(str)):
            if 'A' <= t[i] <= 'Z':
                t[i] = chr(ord(t[i]) + 32)
        return ''.join(t)


s = Solution()
b = "asdosajoOIJSIOJDAHWJEIODSKL   OISDJIOJDSO isjdiojs SIOJODISJ asdajiosSDJIOJOIDS oaisjdoi"
a = s.toLowerCase(b)
print(a)
print(b)

File: 709 - To Lower Case.java
class Solution {
    public String toLowerCase(String str) {
        char[] c = str.toCharArray();
        for (int i = 0; i < c.length; i++) {
            if (Character.isUpperCase(c[i])) {
                c[i] = Character.toLowerCase(c[i]);
            }
        }
        String res = new String(c);
        return res;
    }
}